marketplacetests/marketplace_gaia_test.py

```diff
@@ -29,6 +31,12 @@ def setUp(self):
         # This is used to tell FxA whether to create a dev or prod account
         self.base_url = 'https://marketplace-dev.allizom.org'
 
+    @property
+    def new_account(self):
```

Comment:
Nit: I wouldn't make this a property, I'd make it a method named something like `create_firefox_account`.